### ISSUE TITLE
suggest for rtl810xe [10ec:8136]

### ISSUE THREAD

**Issue by sacarde** [ghostrider-reborn/realtek-r8101-linux-driver#2]
hi,
   I try installing this module r8101 with my card rtl810xe [10ec:8136] but dont works , I only view:

2: enp3s0: <broadcast,multicast,up,lower_up> mtu 1500 qdisc fq_codel state UP group default qlen 1000 link/ether 2b:d3:11:47:76:a6 brd ff:ff:ff:ff:ff:ff

which module suggest me to try? r8169 into kernel? (5.3.0-46-generic) on ubuntu 18.04

thank you

**Comment by sacarde** [comment]
I read in your README.md 

...  Installing the official driver for 810xE series fixes it.

but which is "official driver" ?

in this page? https://www.realtek.com/en/component/zoo/category/network-interface-controllers-10-100-1000m-gigabit-ethernet-pci-express-software

thank you

**Comment by adithya2306** [comment]
> I read in your README.md
> 
> ... Installing the official driver for 810xE series fixes it.
> 
> but which is "official driver" ?
> 
> in this page? https://www.realtek.com/en/component/zoo/category/network-interface-controllers-10-100-1000m-gigabit-ethernet-pci-express-software
> 
> thank you

yes

**Comment by sacarde** [comment]
thanks...

I download and build 
"GBE Ethernet LINUX driver r8168 for kernel up to 4.15"
(r8168-8.048.02) but in "modinfo r8168" I dont view [10ec:8136]...

is this OK, or I wrong tarball?

**Comment by adithya2306** [comment]
> thanks...
> 
> I download and build
> "GBE Ethernet LINUX driver r8168 for kernel up to 4.15"
> (r8168-8.048.02) but in "modinfo r8168" I dont view [10ec:8136]...
> 
> is this OK, or I wrong tarball?

wrong. rtl810xe is 100M FE NIC so u shud use FE Ethernet LINUX driver for kernel up to 4.15 (same as this repo)

**Comment by sacarde** [comment]
ok ok, I view, FE Ethern...ecc

sorry... 

but for 5.3.0-46-generic what do you suggest?

**Comment by bukyapavankumar** [comment]
I have same rtl8loxe ,I tried to follow your guide 
But it doesn't load the driver

**Comment by bukyapavankumar** [comment]
Wher can I contact you @ghostrider-reborn
